### PR TITLE
chore: Add `composer-normalize` to auto-review testing of composer.json

### PR DESCRIPTION
**Pull request by paulbalandan**
**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Comment by paulbalandan** [comment]
composer-normalize requires PHP 8.1, so this would not work on lower PHP when running the workflows

**Review by MGatner — COMMENTED** [review]
I use Normalize in my own projects, very helpful. This is a significant addition to the workflow spec - is it worth making it standalone?

**Comment by kenjis** [comment]
We can use v2.28 on PHP 7.4.
```
$ composer require --dev ergebnis/composer-normalize
Cannot use ergebnis/composer-normalize's latest version 2.33.0 as it requires php ~8.1.0 || ~8.2.0 which is not satisfied by your platform.
./composer.json has been updated
Running composer update ergebnis/composer-normalize
Loading composer repositories with package information
Updating dependencies
Lock file operations: 6 installs, 0 updates, 0 removals
  - Locking ergebnis/composer-normalize (2.28.3)
  - Locking ergebnis/json-normalizer (2.1.0)
  - Locking ergebnis/json-printer (3.2.0)
  - Locking ergebnis/json-schema-validator (2.0.0)
  - Locking justinrainbow/json-schema (5.2.12)
  - Locking localheinz/diff (1.1.1)
```

**Review by kenjis — APPROVED** [review]
LGTM!

File: .github/workflows/test-autoreview.yml [hunk 1]
```diff
@@ -29,3 +29,79 @@ jobs:
...
+          fi
+
+          if [[ ${{ contains(steps.normalize-framework-composer.outputs.FRAMEWORK_COMPOSER_RESULT, 'Successfully normalized') }} == true ]]; then
+            echo 'Framework composer.json is not yet normalized.';
+            exit 1;
+          fi
+
+          if [[ '${{ steps.normalize-starter-composer.conclusion }}' == 'failure' ]]; then
+            echo 'Normalization of starter composer.json encountered a problem.';
+            echo 'Please run it locally: `composer normalize -d admin/framework`';
```
Comment (paulbalandan) [inline .github/workflows/test-autoreview.yml:100]:
```suggestion
            echo 'Please run it locally: `composer normalize -d admin/starter`';
```

**Review by kenjis — APPROVED** [review]

**Pull request merged**